CODING_STANDARDS.md

```diff
@@ -0,0 +1,76 @@
+# Coding standards & convention
+
+We have decided to create and follow some guidelines and best practices during development of Capybara theme. There are a few core goals that we stick to because we belive they improve our cooperation and readability of source code we've created.
```

Comment:
Type: is "belive" and should be "believe"

Reply:
Fixed.